test/page-goto.spec.ts

```diff
@@ -499,3 +499,28 @@ it('should not throw unhandled rejections on invalid url', async ({page, server}
   const e = await page.goto('https://www.youtube Panel Title.com/').catch(e => e);
   expect(e.toString()).toContain('Panel Title');
 });
+
+it('should not crash when RTCPeerConnection is used',(test, { browserName }) => {
+  test.fail(browserName === 'webkit');
```

Comment:
Does it fail on linux/windows too?

Reply:
It passes on Linux for me. But we'll fix it shortly, so no big deal